.github/workflows/buildMobileAndroid.yml

```diff
@@ -0,0 +1,33 @@
+name: Mobile android app CI
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches:
+      - 'develop'
+    paths:
+      - 'composeApp/**'
```

Comment:
Do we **launch workflow** only when there are **changes in the root module**?

It’s also worth building and running tests for modules: `common`, `contract`, maybe for some others that are used in the application?

Reply:
I don't think that changes to 'contract' module should trigger application workflows. It's a different module and if we would like to include changes to this module to CI then I think it should be separate workflow which only builds this module.
About 'common' module there's no reason to add it to triggers because it isn't used in any user applications.